aiven/client/client.py

```diff
@@ -60,20 +69,11 @@ def set_auth_token(self, token):
     def set_ca(self, ca):
         self.session.verify = ca
 
-    def _execute(self, func, method, path, body, params=None):
-        url = self.base_url + path
-        headers = {}
+    def _log_request(self, url, method, body, headers, params=None):
```

Comment:
The `_log_request` and `_log_reponse` are probably unnecessary if the urllib3 logging is used instead